documentation/markup.rst

```diff
@@ -565,6 +565,16 @@ The directives are:
 
             Description of the attribute.
 
+   This directive should be used for defining attributes.::
+
+      .. attribute:: my-attr
+         This attribute `my-attr` should be defined with this directive.
+
+   When referring to an attribute, use the ``:attr:`my-attr``` role::
+
+      :attr:`my-attr`
+         Use this role to refer to the attribute `my-attr`
+
```

Comment:
And let's modify this to use "ham", and roles are usually used inline:
```suggestion
   Refer to attributes with ``:attr:`` role::

      Use the :attr:`ham` attribute to spam the eggs.
```
(Or something else [Pythonesque](https://www.oxfordreference.com/display/10.1093/oi/authority.20110803100356667) :)